sdk/agrifood/agrifood-farming-rest/swagger/README.md

```diff
@@ -21,8 +21,8 @@ rest-level-client: true
 security: AADToken
 security-scopes: https://farmbeats.azure.net/.default
 use-extension:
-  "@autorest/typescript": "latest"
-
+  "@autorest/typescript": "6.0.34"
```

Comment:
should we use "latest" for all by default, and only have individual packages to opt to a specific version if there's any block issue?

Reply:
We can! I think I saw more instances where a specific version was pinned to as opposed to using `latest` - I guess one concern might be "as a partner I am using a new service version to generate my SDK, and am getting a bunch of unrelated codegen changes due to a newer version being released.

That may not be a bad thing, just a mildly confusing thing. Any thoughts? If you still think `latest` is the better default I am happy to change it 👍

Reply:
> I guess one concern might be "as a partner I am using a new service version to generate my SDK, and am getting a bunch of unrelated codegen changes due to using a newer version of @autorest/typescript."

Good point! Pinning is actually better.